### DISCUSSION TITLE
Settings for better results

### DISCUSSION THREAD

**Discussion by StableDB** [TheLastBen/fast-stable-diffusion#2249] [Q&A]
Hi everyone, I am trying to train my own face with 26 images at 768x768 and 512x512. But both of them were under my expectations, am I doing something wrong? 
My workflow:
1- Taking good pictures and cropping to sizes that I want to train
2- Renaming all images (myphotosol(1), myphotosol(2)........)  !example!
3- From civit.ai I am selecting one good model to train with
4- Going into colab and naming session name to myphotosol  
5- Make Smart_Crop_images blank then upload my cropped images
6- In this case setting up UNet_Training_Steps: 100x26+500 = 3100 , Save_Checkpoint_Every: 300, Start_saving_from_the_step: 2600
7- Then start training

If I wanted to use myself as ohwx do I have to change image names to ohwx? 
Is session_name important to results, do I have to name it the same as with pictures?
Min how many pictures do I have to use for good modeling?
What are the best settings at the Dreambooth section?

**Comment by THX1139b** [comment]
Seems like people are having better results with fewer photos these days for some reason, around 10 photos. That is what works best for me now but my very best models were trained on a lot more photos around six months ago.
The instructions says to name them myphotosol (1), with a space. But when uploading them the space is then replaced with a - which confused me a bit. But I do include the space myself. Your examples have no space.
My understanding is that the session name should also be the name in the images and that this will be your keyword name. So in your case the photos should be named ohwx (1).png etc and also your session named ohwx.
Lately I have typically used 1800 steps for 10 images, starting to save versions every 200 steps from earlier on. I often check it and continue training beyond that as well, keeping a copy of my output model before the resumed training changes it.
